src/Beberlei/Metrics/Collector/InlineTaggableGaugeableCollector.php

```diff
@@ -1,64 +0,0 @@
-<?php
-/**
- * Beberlei Metrics.
- *
- * LICENSE
- *
- * This source file is subject to the MIT license that is bundled
- * with this package in the file LICENSE.txt.
- * If you did not receive a copy of the license and are unable to
- * obtain it through the world-wide-web, please send an email
- * to kontakt@beberlei.de so I can send you a copy immediately.
- */
-namespace Beberlei\Metrics\Collector;
-
-/**
- * InlineTaggableGaugeableCollector interface.
- */
-interface InlineTaggableGaugeableCollector
```

Comment:
You can not remove an interface, that's a BC break.

I did not look at the code for now, but If you have to do it, you have to depreciate it and provide an upgrade path.

But honestly, I prefer to not remove it to not break the BC.